exercises/square-root/instructions.md

```diff
@@ -0,0 +1,18 @@
+# Instructions
+
+Your task is to calculate the square root of a given number.
+
+- Try to avoid using the pre-existing math libraries of your language.
+- As input you’ll be given a positive whole number, i.e. 1, 2, 3, 4…
+- You also only need to output a positive whole number.
```

Comment:
Is the point here that we're getting the nearest whole number? (ie approximating/rounding?)

Reply:
Yes.  This was chosen to narrow the scope of the exercise and hopefully make it both easier and more interesting.  It also opens the possibility of more alternative implementations such as linear and binary search.